content/md/en/docs/fundamentals/light-clients-in-substrate-connect.md

```diff
@@ -0,0 +1,273 @@
+---
+title:  Light clients and Substrate Connect
+description: Use a light client and Substrate Connect to enable your applications to interact with any Substrate-based chain.
```

Comment:
This is same as said in closed PR. This is meant to replace substrate connect documentation, so IMO it would be better to have something like:
```suggestion
title:  Substrate Connect
description: Use Substrate Connect, which integrates Smoldot light client,  to enable your applications to interact with any Substrate-based chain.
```